src/components/pages/order/adminPanel/BlocTabs.js

```diff
@@ -0,0 +1,99 @@
+import styled from "styled-components";
+import Tab from "../../reusable-ui/Tab";
+import { BsChevronDown, BsChevronUp } from "react-icons/bs";
+import { AiOutlinePlus } from "react-icons/ai";
+import { RiPencilFill } from "react-icons/ri";
+import { theme } from "../../../../theme";
+import { useContext } from "react";
+import GlobalContext from "../../../../context/GlobalContext";
+import BlocAdminPanelContent from "./BlocAdminPanelContent";
+
+export default function BlocTabs() {
+  const { isPanelReduce, panelTabIndex, setPanelTabIndex, setIsPanelReduce } =
+    useContext(GlobalContext);
+
+  // management of the display of different content according to the index
+  const toggleTab = (index) => {
+    setPanelTabIndex(index);
+    if (isPanelReduce === true) {
+      setIsPanelReduce(false);
+    }
+  };
+
+  const hideAndShowAdminPanelContent = () => {
+    setIsPanelReduce(!isPanelReduce);
+  };
+  return (
+    <>
+      <BlocTabsStyled>
+        <Tab
+          text={""}
+          className={`tabs tabs-one ${panelTabIndex === 1 && "is-active-tab"}`}
+          Icon={isPanelReduce ? <BsChevronUp /> : <BsChevronDown />}
+          onClick={() => hideAndShowAdminPanelContent()}
```

Comment:
```suggestion
          onClick={hideAndShowAdminPanelContent}
```

Puisque tu n'envoies pas de paramètres / variables à hideAndShowAdminPanelContent

Reply:
effectivement